OpenSubtitlesHandler/Models/ApiResponse.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+
+namespace OpenSubtitlesHandler.Models
+{
+    public class ApiResponse<T>
+    {
+        public HttpStatusCode Code { get; }
+        public string Body { get; } = string.Empty;
```

Comment:
Is this accessed anywhere?

Reply:
Yes, [here](https://github.com/MBR-0001/jellyfin-plugin-opensubtitles/blob/master/Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs#L213-L230), [here](https://github.com/MBR-0001/jellyfin-plugin-opensubtitles/blob/master/Jellyfin.Plugin.OpenSubtitles/API/OpenSubtitlesController.cs#L45-L51) and in error logging